app/views/publishers/statements/index.html.slim

```diff
@@ -9,31 +9,8 @@ noscript
   )
   = render partial: "choose_channel_button"
 
-/ .single-panel--wrapper.single-panel--wrapper--xlarge#statements_panel
```

Comment:
Would you see if `app/assets/stylesheets/pages/statements.scss` is removable too?

Reply:
It was, fixed.